db/007-rename_location_human.sql

```diff
@@ -0,0 +1,21 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 7;
```

Comment:
Depending on who merges first, I think yours might end up being `migrationId` 8 and similar change in the file name.

Reply:
I asked Mario about this yesterday and he said to make it 7 and he would name his 8

Reply:
To add more info, mine has been run as 7 on the dev db as discussed here: https://bravely.slack.com/archives/CE32NB85N/p1608154097243000